pages/about.html

```diff
@@ -5,8 +5,47 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link rel="stylesheet" href="../styles/about.css">
+  <link rel="stylesheet" href="../styles/styles.css">
 </head>
 <body>
+  <header>
+    <h1><a href="index.html">Camila Tagle</a></h1>
+      <nav id="menu_bar">
+        <ul>
+          <li><a href="about.html">About me</a></li>
+          <li><a href="portfolio.html">Projects and experience</a></li>
+          <li><a href="future.html">Will Do</a></li>
+        </ul>
+      </nav>
+  </header>
+<ul id="aboutme">
+  <li><h3>My family</h3>
```

Comment:
In semantic HTML, heading tags should be used to indicate different sections of content, not as stylistic elements. Avoid skipping numbers: do `h1` at the highest level, then `h2`, etc.   [Link](https://bighack.org/how-to-heading-structure-accessibility/)